### ISSUE TITLE
Overview Scrollbar

### ISSUE THREAD

**Issue by tokejepsen** [pyblish/pyblish-qml#180]
**Goal**

To show/hide a scrollbar in the Overview when the list of instance and plugins get too long to show.

**Motivation**

Users often don't pick up on that the list of instances and plugins are longer than shown, so we need some visual indicator.

**Implementation**

@mottosso 
> Ok, there's a [scrollbar](https://github.com/pyblish/pyblish-qml/blob/b879e2813959b7b4111f3caa9e5728163218a5c8/pyblish_qml/qml/Perspective/Viewport.qml#L117) implemented for the Perspective view (when you "enter" an item).
It should be easily plug-and-playable with the ListView used in the Overview. Could we open a separate issue for this, so we can properly close it once implemented?

**Comment by mottosso** [comment]
> show/hide

Oh, didn't think of that.

If we can get a menubar into the GUI, we could have a whole new level of customisability where everyone gets what they want. :) Another issue? :)

**Comment by mottosso** [comment]
What do you think about having scrollbars appear only when the user hovers one of the lists? For example, if the user hovers the plug-ins list, a scrollbar appears.

And not instantly, but via an ease-in animated horizontal scale of say 0.2 seconds.

That way, the GUI might remain minimal where it counts, and help out only where it needs to.

**Comment by tokejepsen** [comment]
> What do you think about having scrollbars appear only when the user hovers one of the lists?

Yeah, sounds good to me. I was going to suggest the scroll bar that you have in pyblish-tray as well to keep it minimal, but show/hiding depending on hovering might be even better.

**Comment by mottosso** [comment]
Could do both!

**Comment by tokejepsen** [comment]
Just had a further idea for this. To indicate to begin with, that there are more items in the list, we could fade the text. Seen this effect in multiple places now.

![capture1](https://user-images.githubusercontent.com/1860085/32044918-34ff54da-ba3f-11e7-98d3-7d3046fd5918.PNG)

**Comment by mottosso** [comment]
Looks good to me. It's easy to implement too.

**Comment by darkvertex** [comment]
Hey guys, any update on this?

I have artists using Pyblish-based publishers getting a little confused by the lack of a scrollbar sometimes.

**Comment by tokejepsen** [comment]
Dont think anyone has or are looking into this. PR is welcome :)

**Comment by darkvertex** [comment]
QML is unfortunately alien technology to me at the moment. I wish I knew it better. :/

I suppose there's some sort of "layout" somewhere missing an attribute to enable the scrollbar? (Am I getting warm?)

**Comment by tokejepsen** [comment]
QML can be quite daunting, but it pays off in the end :)

The area you would need to look into starts here https://github.com/pyblish/pyblish-qml/blob/master/pyblish_qml/qml/Overview.qml#L75.
